frontend/app/src/store/history/actions.ts

```diff
@@ -509,12 +514,22 @@ export const actions: ActionTree<HistoryState, RotkehlchenState> = {
 
     setStatus(refresh ? Status.REFRESHING : Status.LOADING);
 
-    try {
-      const { taskId } = await api.history.ledgerActions();
+    const locations = balances!.connectedExchanges
+      .map(({ location }) => location)
+      .filter(location => location in SUPPORTED_EXCHANGES)
```

Comment:
> * Ledger Actions not loading unless SUPPORTED_EXHANGE coinbase is connected

There is an open PR to return the locations that have data but are not connected exchanges, but it is not complete yet. Unfortunately, at the moment there is no way to know about these locations. 

For other sources like some of the CSV imports that add external trades, the frontend has hardcoded values, which is sub-optimal but it's the only viable solution at the moment.